.github/workflows/workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: workflow
+on: [pull_request]
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go 1.14
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Lint code
+        uses: golangci/golangci-lint-action@e4e910b0cea5459cf5f459ac51f61ebb51934e36
+        with:
+          version: v1.36.0
+          skip-go-installation: true
```

Comment:
By default, the linter will install and use some version of Go 1 (it happens to be 1.15.x) rather than honor the one that we've manually chosen. One unintended side effect is that all subsequent steps will use this other version of Go, which is not what we want. A recent commit in December added an option to skip the installation but unfortunately this hasn't made it into a release so we're using that specific commit directly.

Reply:
Yeah, I like that we're using the same version of Go (the one we select) for everything. Also, specifying exact SHA of a GH Action step/lib is actually better than using releases, which can be modified by 3rd parties.